importer.js

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import nodePath from 'path';
+import csvjson from 'csvjson';
+import { promisify } from 'util';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+class Importer {
+    constructor(dirWatcher) {
+        dirWatcher.on('changed', async (path) => {
+            /**
+             * Watching for file changes Syncrhonously
+             */
+            let syncData = this.importSync(path);
+            console.log(result)
+
+            /**
+             * Asynchronously watching the file changes
+             */
+            let asyncData = await this.import(path);
+            console.log(asyncData); 
+        });
+    }
+
+    async import(path) {
+        try {
+            let jsonData = [], newFileData = [];
+            let fileNames = await readDirAsync(path);
+            for (const fileName of fileNames) {
+                let data = await readFileAsync(nodePath.join(__dirname, path, fileName), { encoding: 'utf8' });
+                newFileData = csvjson.toObject(data);
+                jsonData = jsonData ? [...jsonData, ...newFileData] : newFileData;
+            }
+
+            return jsonData;
+        } catch (error) {
+            console.err(error);
+        }
+    }
+
+    importSync(path) {
+        let jsonData, newFileData;
+        fs.readdirSync(path).forEach(fileName => {
+            let data = fs.readFileSync(nodePath.join(__dirname, path, fileName), { encoding: 'utf8' });
```

Comment:
`const` is preferable since `data` variable is assigned with value only once when the `importSync()` method is called

Reply:
Added `const `over `let`